build/Build.Settings

```diff
@@ -84,4 +84,10 @@
      </Link>
    </ItemDefinitionGroup>
  
+  <PropertyGroup>
+    <_TwoDigitYear>$([MSBuild]::Subtract($([System.DateTime]::UtcNow.Year), 2000))</_TwoDigitYear>
+    <_ThreeDigitDayOfYear>$([System.DateTime]::UtcNow.DayOfYear.ToString().PadLeft(3, '0'))</_ThreeDigitDayOfYear>
+    <AssemblyRevision>$(_TwoDigitYear)$(_ThreeDigitDayOfYear)</AssemblyRevision>
```

Comment:
So, when we produce more than one build in the same day, won't this lead to duplicate file revisions?

Reply:
Yeah it does. Were we okay with doing this for other dlls because we use nuget package versions?

Reply:
AFAIK, we use this pattern elsewhere

Reply:
Where do we use this? It means our builds are not reproducible, and that two builds with different sources will have the same revision number. Although it may not be a huge problem, it's certainly not a good thing.

Reply:
I think it is used for all of our dlls today...

Reply:
Other dotnet dlls have the file version matching the product version.

Reply:
Ok. This is still bad for reproducible builds. But it is bad in the same way the rest of our stack computes file version, so we can just add this to the list of offenses to be cleaned up when we fix https://github.com/aspnet/BuildTools/issues/452